torch/autograd/gradcheck.py

```diff
@@ -191,13 +193,13 @@ def compute(a, b):
     return tuple(compute(a, b) for (a, b) in zip(outa, outb))
 
 
-def compute_numerical_jacobian_cols(jvp_fn, delta, input_is_complex) -> List[torch.Tensor]:
+def _compute_jvps_wrt_specific_input(jvp_fn, delta, input_is_complex) -> List[torch.Tensor]:
```

Comment:
You want to keep the `numerical` vs `analytical` in these new namings.
Mostly because soon, we will add an `analytical` jvp function here.